selectors.yml

```
@@ -107,18 +152,30 @@ selectors:
   421614:
     selector: 3478487238524512106
     name: "ethereum-testnet-sepolia-arbitrum-1"
+  432201:
+    selector: 1458281248224512906 
+    name: "avalanche-subnet-dexalot-testnet"
   534351:
     selector: 2279865765895943307
     name: "ethereum-testnet-sepolia-scroll-1"
+  686868:
+    selector: 5269261765892944301 
+    name: "bitcoin-testnet-merlin"
   595581:
     selector: 7837562506228496256
     name: "avalanche-testnet-nexon"
+  810181: 
+    selector: 5837261596322416298
+    name: "zklink-nova-testnet" 
```

For consistency with zklink_nova-mainnet:
```suggestion
    name: "zklink_nova-testnet" 
```